File: Task-03/C/subtask2.c
#include <stdio.h>
int main()
{ FILE *inputFile, *outputFile;
    char buffer[1000]; 
    inputFile = fopen("input.txt", "r");
    if (inputFile == NULL) {
        printf("Error opening input.txt\n");
        return 1;
    }
    outputFile = fopen("output.txt", "w");
    if (outputFile == NULL) {
        printf("Error opening output.txt\n");
        fclose(inputFile);  
        return 1;
    }
    while (fgets(buffer, sizeof(buffer), inputFile) != NULL) {
        fputs(buffer, outputFile);
    }

    /
    fclose(inputFile);
    fclose(outputFile);

    printf("Content has been successfully copied from input.txt to output.txt\n");

    return 0;
}

File: Task-03/Rust/subtask2.rs
use std::fs;
use std::io::Result;

fn main() -> Result<()> {
    
    let input_path = "input.txt";
    let output_path = "output.txt";

    let content = fs::read_to_string(input_path)?;

    fs::write(output_path, content)?;

    println!("String has been read from {} and writen to {}.", input_path, output_path);

    Ok(())
}

